src/utils/evalUtils.js

```diff
@@ -239,45 +36,6 @@ function prepareTransformArgs(generator) {
   return args;
 }
 
-/**
- * Safely executes the toUrl function with the arguments provided.
- *
- * @param {String} functionBody - The toUrl function body as provided by the
- *  sample generator template.
- * @param {Object} args - An object containing the following attributes:
- *  {Object} ctx - The sample generator context.
- *  {Array} aspects - Array of one or more aspects.
- *  {Array} subjects - Array of one or more subjects.
- * @returns {String} - The generated url as a string
- * @throws {ToUrlError} - if transform function does not return an array
- *  of zero or more samples
- * @throws {ArgsError} - if thrown by validateToUrlArgs function
- * @throws {FunctionBodyError} - if thrown by safeEval function or if function
- *  body is not a string
- */
-function safeToUrl(functionBody, args) {
-  debug('Entered evalUtils.safeToUrl', args);
-  if (typeof functionBody !== 'string') {
-    const msg = 'toUrl function body must be a string';
-    throw new errors.FunctionBodyError(msg);
-  }
-
-  validateToUrlArgs(args);
-  const retval = safeEval(functionBody, args);
-  if (typeof retval !== 'string') {
-    throw new errors.ToUrlError(ERROR_MESSAGE.TO_URL.NOT_STRING);
-  }
-
-  debug('evalUtils.safeToUrl returning: ${retval}');
-  return retval;
-}
-
 module.exports = {
-  safeEval, // exporting for testability
-  safeToUrl,
-  safeTransform,
   prepareTransformArgs,
```

Comment:
If this is the only method left, we should just move it to handleCollectResponse and get rid of this file.

Reply:
ok